docs/source/array-creation.rst

```diff
@@ -64,7 +64,7 @@ Often we store data in several different locations and want to stitch them toget
         x = da.from_array(d, chunks=(1000, 1000))
         dask_arrays.append(x)
 
-    x = da.concatenate(arrays, axis=0)  # concatenate arrays along first axis
+    x = da.concatenate(dask_arrays, axis=0)  # concatenate arrays along first axis
```

Comment:
Not blocking, but maybe we should rename `x` here to something else (given it is used in the `for`-loop above). Alternatively we could rename the `x` in the `for`-loop.

Reply:
Good point. I can change the `x` inside the `for`-loop to `array`. I.e.

```python
dask_arrays = []
for fn in filenames:
    f = h5py.File(fn)
    d = f['/data']
    array = da.from_array(d, chunks=(1000, 1000))
    dask_arrays.append(array)

x = da.concatenate(dask_arrays, axis=0)
```

Reply:
SGTM